cmd/super/root/command.go

```diff
@@ -24,35 +24,39 @@ import (
 
 var Super = &charm.Spec{
 	Name:        "super",
-	Usage:       "super [options] <command> | super [ options ] [ -c [ spq/sql ] ] [ file ... ]",
-	Short:       "process data with Zed queries",
+	Usage:       "super [options] <command> | super [ options ] [ -c query ] [ file ... ]",
+	Short:       "process data with SuperSQL queries",
 	HiddenFlags: "cpuprofile,memprofile,pathregexp",
 	Long: `
-XXX "super" is a command-line tool for processing data in diverse input formats,
+The "super" command provides a way to process data in diverse input formats,
 providing search, analytics, and extensive transormations using
-the Zed query language.
+the SuperSQL query language.
 A query typically applies Boolean logic or keyword search to filter
 the input and then transforms or analyzes the filtered stream.
 Output is written to one or more files or to standard output.
 
-A Zed query is comprised of one or more operators interconnected
+A query is comprised of one or more operators interconnected
 into a pipeline using the Unix pipe character "|".
```

Comment:
```suggestion
into a pipeline using the Unix pipe character "|>".
```

Want to aspirationally make this change now on the assumption #5427 will go through?

BTW, I briefly paused on what to call `|>` since it's not a single character, but I see the Google paper still calls it " the pipe character '|>'" so it seems like we can just go with that (though I also think the Unix comparison is still totally worth keeping).

Reply:
using the pipe symbol "|>"